assets/prototype/application/services/apollo/mutations/useEntityMutation.ts

```diff
@@ -133,7 +150,13 @@ const useEntityMutation = (type: EntityType, id?: string): EntityMutation => {
 	/**
 	 *
 	 */
-	const onMutationComplete = (response: FetchResult, onCompleted: OnMutationCompletedFn): void => {
+	const onMutationComplete = (
+		response: FetchResult,
+		onCompleted: OnMutationCompletedFn,
+		mutationType: MutationType
+	): void => {
+		toaster.dismiss(getToasterMessage(mutationType));
```

Comment:
Maybe I am too picky, but what about:
```
const dismissMessage = getToasterMessage(mutationType);
const successMessage = `successfully ${getToasterMessage(mutationType, 'ed')}`;
toaster.dismiss(dismissMessage);
toaster.success(successMessage);
```
***
this is optional :point_up: 
you can ignore it :)